external-api/src/database/mongo-client.ts

```diff
@@ -1,34 +1,11 @@
-const { MongoClient } = require('mongodb'); // eslint-disable-line @typescript-eslint/no-var-requires
+import { MongoDbClient, MongoDbCollectionName } from '@ukef/dtfs2-common';
 import { dbName, url } from '../config';
 
-let client: any;
-let connection: any = null;
+const mongoDbClient = new MongoDbClient({ dbName, dbConnectionString: url });
```

Comment:
Are we doing this here so that the common package doesn't have to know the .env files? It feels like this whole file is a bit of a duplicate of the common DB and wondering if we can just have the common DB file and get rid of these files in each of the microservices. What are your thoughts?

Reply:
I originally wanted to do this, but Mike wanted to avoid needing to define the MongoDB connection `.env` vars in common. That being said, I need them anyway for the seeder, so I might centralise the initialisation of the `MongoDbClient`